helper/validation/validation.go

```diff
@@ -339,3 +339,22 @@ func FloatBetween(min, max float64) schema.SchemaValidateFunc {
 		return
 	}
 }
+
+// StringDoesNotContainAny returns a SchemaValidateFunc which validates that the
+// provided value does not contain any of the specified Unicode code points in chars.
+func StringDoesNotContainAny(chars string) schema.SchemaValidateFunc {
```

Comment:
My only point of debate is if the func name should be more explicit like `StringDoesNotContainAnyChars()` 🤔 . Thoughts @radeksimko @kmoe ? Overall though approve of the feature, holding off merge until we are all in agreement on the func name. (I'm not particularly torn either way)

Reply:
I agree that the name could be improved, but perhaps from a different aspect.

In regards to `...ContainAny` vs `...ContainAnyChars` or `...ContainChars` - I understand your concerns that there could be a confusion whether the function accepts `[]string`, `string` or even `[]byte`, but I would just bet on Go here, which apparently picked `strings.ContainsAny`. Admittedly whole `strings` library conveys the message that `string` will be the overwhelmingly used type, unlike here. So I don't have a strong opinion in this regards - just explaining my train of thoughts.

What concerns me more is the negative in its name/logic. If we can come up with an easy way to negate a validation function we could just turn this into `StringContainsChars()`.

So end-user would end up with something like `ValidateFunc: validation.Not(validation.StringContainsAny(""))` which IMO looks more natural and also kills another bird with the same stone - we get validation function for the opposite.

Reply:
I think I chose `StringDoesNotContainAny()` as that matched the name of the golang function `strings.ContainsAny()` which is used in the implementation. No problems changing it to whatever is agreed upon.

Reply:
I see the consistency now. @radeksimko I am exploring if we can add a negation feature but I don't think it is easily possible as some error checks are fatal in both scenarios such as checking the type (which we see in this PR).